src/Rings/NumberField.jl

```diff
@@ -743,8 +743,13 @@ end
 
 function Hecke.any_order(K::NfNSGen)
   B = basis(K, copy = false)
-  for b in B
-    @assert isone(denominator(b.f))
+  B = copy(B)
```

Comment:
Depending on the implementation of `basis`, it may be more efficient to call `B = basis(K; copy=true)` instead of explicitly calling `copy`